HousingRegisterApi/V1/Functions/NovaletExportGeneratorFunction.cs

```diff
@@ -0,0 +1,62 @@
+using Amazon.Lambda.Core;
+using HousingRegisterApi.V1.Gateways;
+using HousingRegisterApi.V1.Infrastructure;
+using HousingRegisterApi.V1.Services;
+using HousingRegisterApi.V1.UseCase;
+using HousingRegisterApi.V1.UseCase.Interfaces;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace HousingRegisterApi.V1.Functions
+{
+    public class NovaletExportGeneratorFunction : BaseFunction, ILambdaFunctionHandler
+    {
+        /// <summary>
+        /// Default constructor. This constructor is used by Lambda to construct the instance. When invoked in a Lambda environment
+        /// the AWS credentials will come from the IAM role associated with the function and the AWS region will be set to the
+        /// region the Lambda function is executed in.
+        /// </summary>
+        public NovaletExportGeneratorFunction()
+        {
+
+        }
+
+        public void Handle(ILambdaContext context)
+        {
+            context.Logger.LogLine("Generating Novalet export for approval");
+            var getExportFileUseCase = ServiceProvider.GetService<IGetNovaletExportUseCase>();
```

Comment:
Can these be injected?

Reply:
don't think so as this is the entry point